src/libraries/Microsoft.PowerFx.Core/Public/Values/TableValue.cs

```diff
@@ -181,5 +183,28 @@ public override void Visit(IValueVisitor visitor)
         {
             visitor.Visit(this);
         }
+
+        public override string ToExpression()
+        {
+            if (Rows.Count() == 0)
+            {
+                throw new InvalidDataContractException("It's not possible to serialize an empty table.");
+            }
+
+            var sb = new StringBuilder();
+
+            sb.Append("Table(");
+
+            foreach (var row in Rows)
+            {
+                sb.Append(row.Value.ToExpression());
+                sb.Append(",");
+            }
+
+            sb.Remove(sb.Length - 1, 1);
```

Comment:
>sb.Remove(sb.Length - 1, 1); [](http://example.com/codeflow?start=12&length=28)

Not a big fan

Reply:
yes, definitely prefer [String.Join](https://learn.microsoft.com/en-us/dotnet/api/system.string.join?view=net-6.0)

Reply:
or use a flag to avoid the sb.Append(,) on the first iteration.